docs/sp/lists.md

```diff
@@ -0,0 +1,487 @@
+
+// log resulting item array to console
+console.log(r);
+```
+
+### Get list items changes using a Token
+
+```TypeScript
+import {  IChangeLogItemQuery } from "@pnp/sp/src/lists";
+
+// build the caml query object (in this example, we include Title field and limit rows to 5)
+const changeLogItemQuery: IChangeLogItemQuery = {
+    Contains: `<Contains><FieldRef Name="Title"/><Value Type="Text">Item16</Value></Contains>`,
+    QueryOptions: `<QueryOptions>
+    <IncludeMandatoryColumns>FALSE</IncludeMandatoryColumns>
+    <DateInUtc>False</DateInUtc>
+    <IncludePermissions>TRUE</IncludePermissions>
+    <IncludeAttachmentUrls>FALSE</IncludeAttachmentUrls>
+    <Folder>My List</Folder></QueryOptions>`,
+};
+
+// get list items
+const r = await list.getListItemChangesSinceToken(changeLogItemQuery);
+
+// log resulting XML to console
+console.log(r);
+```
+
+### Recycle a list
+
+Removes the list from the web's list collection and puts it in the recycle bin.
+
+```TypeScript
+await list.recycle();
+```
+
+### Render list data
+
+```TypeScript
+import { IRenderListData } from "@pnp/sp/src/lists";
+
+// render list data, top 5 items
+const r: IRenderListData = await list.renderListData("<View><RowLimit>5</RowLimit></View>");
+
+// log array of items in response
+console.log(r.Row);
+```
+
+### Render list data as stream
+
+```TypeScript
+import { IRenderListDataParameters } from "@pnp/sp/src/lists";
+
+// setup parameters object
+const renderListDataParams: IRenderListDataParameters = {
+    ViewXml: "<View><RowLimit>5</RowLimit></View>",
+};
+
+// render list data as stream
+const r = await list.renderListDataAsStream(renderListDataParams);
+
+// log array of items in response
+console.log(r.Row);
+```
+
+### Reserve list item Id for idempotent list item creation
+
+```TypeScript
+const listItemId = await list.reserveListItemId();
+
+// log id to console
+console.log(listItemId);
+```
+
+### Get list item entity type name
+
+```TypeScript
+const entityTypeFullName = await list.getListItemEntityTypeFullName();
+
+// log entity type name
+console.log(entityTypeFullName);
+```
+
+### Add a list item using path (folder), validation and set field values
+
+```TypeScript
+const list = await sp.webs.lists.getByTitle("MyList").select("Title", "ParentWebUrl").get();
+const formValues: IListItemFormUpdateValue[] = [
+                {
+                    FieldName: "Title",
+                    FieldValue: title,
+                },
+            ];
+            
+list.addValidateUpdateItemUsingPath(formValues,`${list.ParentWebUrl}/Lists/${list.Title}/MyFolder`)
+
+```
+## content-types imports
+
+|Scenario|Import Statement|
+|--|--|
+|Selective 1|import "@pnp/sp/src/content-types";|
+|Selective 2|import "@pnp/sp/src/content-types/list";|
+|Preset: All|import { sp } from "@pnp/sp/presents/all";|
+
+### contentTypes
+
+Get all content types for a list
+```TypeScript
+const list = await sp.web.lists.getByTitle("Documents");

```

Comment:
```suggestion
const list = sp.web.lists.getByTitle("Documents");
```  

I think `await` isn't needed here, because `sp.web.lists.getByTitle("Documents")` is not awaitable (i.e. it doesn't return a Promise instance), instead it returns a proxy object.    

`sp.web.lists.getByTitle("Documents").get()` (pnpjs v1 style) and `sp.web.lists.getByTitle("Documents")()` (new v2 style) are awaitable. Probably there are some other places in docs with the same pattern (that's the one I spotted).